content/syllabuses/java.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Java Basics 
+---
+
+- Java Collections
+- {{% contentlink "topics/java-specific/collections-and-datastructures" %}}
+- {{% contentlink "projects/java-specific/collections" %}}
+- Java Data Structures
+- {{% contentlink "topics/java-specific/collections-and-datastructures" %}}
+- {{% contentlink "projects/java-specific/data-structures" %}}
+- Java Generics
+- {{% contentlink "topics/java-specific/generics" %}}
+- {{% contentlink "projects/java-specific/generics" %}}
```

Comment:
Doesnt the project have the topic as a prerequisite? 
I would remove this line: 
- {{% contentlink "topics/java-specific/generics" %}}

Reply:
implemented across all.